### PR TITLE
Adjust Github Actions

### PR DESCRIPTION
**Pull request by dralley**
[noissue]

### CODE REVIEW

**Comment by fao89** [comment]
from the logs I see it failing here:
https://github.com/pulp/pulp-openapi-generator/blob/master/generate.sh#L14
https://github.com/pulp/pulp-openapi-generator/pull/38/checks?check_run_id=1486869189#step:8:24
I believe `httpie` is expecting a `config.json` that does not exist:
https://github.com/pulp/pulp-openapi-generator/blob/master/.github/workflows/ci.yml#L105
`config.json` like this:
https://github.com/pulp/pulp_ansible/blob/master/.ci/assets/httpie/config.json

**Comment by dralley** [comment]
That might be an issue too but it looks like the main problem is this.

```
line 1032, in _create_direct_connection\n    client_error=client_error,\n  File \"/usr/local/lib64/python3.7/site-packages/aiohttp/connector.py\", line 975, in _wrap_create_connection\n    raise client_error(req.connection_key, exc) from exc\n","description":"Cannot connect to host repos.fedorapeople.org:443 ssl:default [Connect call failed ('152.19.134.199', 443)]"}
```

File: .github/workflows/scripts/script.sh [hunk 1]
```diff
@@ -10,65 +10,37 @@
...
-  gem build pulpcore_client
-  gem install --both ./pulpcore_client-0.gem
-  cd ..
-  rm -rf ./pulp_file-client
+./generate.sh pulp_file ruby 0
 
-  ./generate.sh pulp_file ruby 0
+cd pulp_file-client
+gem build pulp_file_client
+sudo gem install --both ./pulp_file_client-0.gem
```
Comment (fao89) [inline .github/workflows/scripts/script.sh:42]:
```suggestion
gem install --both ./pulp_file_client-0.gem
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -86,7 +59,7 @@ jobs:
           sudo apt-get update -yq
           sudo -E apt-get -yq --no-install-suggests --no-install-recommends install httpie
           echo ::endgroup::
-          echo "TEST=${{ matrix.env.TEST }}" >> $GITHUB_ENV
+          echo "HTTPIE_CONFIG_DIR=$GITHUB_WORKSPACE/.ci/assets/httpie/" >> $GITHUB_ENV
```
Comment (fao89) [inline .github/workflows/ci.yml:62]: Line 78 is overwriting this, please delete line 78

File: .github/workflows/scripts/test_bindings.rb [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+      created_resources = monitor_task(upload_commit_response.task)
+      @artifacts_api.read(created_resources[0])
+    end
+end
+
+
+artifact = upload_file_in_chunks(File.join(ENV['GITHUB_WORKSPACE'], 'README.rst'))
+
+# Create a File Remote
+remote_url = 'https://repos.fedorapeople.org/pulp/pulp/demo_repos/test_file_repo/PULP_MANIFEST'
```
Comment (fao89) [inline .github/workflows/scripts/test_bindings.rb:106]:
```suggestion
remote_url = 'https://fixtures.pulpproject.org/file/PULP_MANIFEST'
```

Comment (dralley) [reply .github/workflows/scripts/test_bindings.rb:106]: I'm going to let https://github.com/pulp/pulpcore/pull/1049 merge first and then copy the files again

**Review by fao89 — APPROVED** [review]
Thank you @dralley

**Pull request merged**